drivers/serial/uart_esp32.c

```diff
@@ -328,6 +330,13 @@ static int uart_esp32_configure(const struct device *dev, const struct uart_conf
 
 	uart_hal_set_rx_timeout(&data->hal, 0x16);
 
+	if(config->tx_invert){
```

Comment:
I think these 2 ' if' cases are missing code style.. that will also fail. Please, fix it.